### DISCUSSION TITLE
Elastic Net using only precomputed matrices

### DISCUSSION THREAD

**Discussion by tzachar** [intel/scikit-learn-intelex#1635] [Q&A]
Hello.

I am trying to utilize daal4py to replace sklearn.
My data comprises only of the gram matrix (xTx) and the precomputed xTy
I have been using `cd_fast.enet_coordinate_descent_gram` from the `_cd_fast` cython module in sklearn to achieve this. However, I have no idea how to accomplish something similar using daal4py.

Any ideas?

**Comment by syakov-intel** [comment]
This is currently not possible. Sorry for the late response.
